crates/holochain_zome_types/src/signature.rs

```diff
@@ -43,30 +49,66 @@ impl Sign {
 }
 
 /// The raw bytes of a signature.
-#[derive(Clone, Serialize, Deserialize, SerializedBytes, Hash, PartialEq, Eq, PartialOrd, Ord)]
-pub struct Signature(#[serde(with = "serde_bytes")] pub Vec<u8>);
+#[derive(Clone, PartialOrd, Hash, Ord)]
+/// The equality is not different, it's just constant time, so we can derive a hash.
+/// For an actually secure thing we wouldn't want to just assume a safe default hashing
+/// But that is not what clippy is complaining about here.
+#[allow(clippy::derive_hash_xor_eq)]
+pub struct Signature(pub [u8; SIGNATURE_BYTES]);
```

Comment:
Does serialize handle this properly without the `serde_bytes`?

Reply:
yup